### ISSUE TITLE
How to send email via gmail?

### ISSUE THREAD

**Issue by ZedZipDev** [myloveCc/NETCore.MailKit#15]
Is it possible to send email via gmail.com? What are settings?
Thnx.

**Comment by zizzo81** [comment]
Yes it is, but you don't enter you email address as username and your password, you will have to create a specific password for "a less secure application" and use that one. Also, business accounts (GSuite) could be prevented using this and accounts that have multi factor authentication enabled cannot enable access to less secure applications
Just use smtp.google.com as your SMTP server with port 465 and SSL enabled.
